apps/action_data_fetcher/lib/action_data_fetcher/gpo/parser/worker.ex

```diff
@@ -0,0 +1,63 @@
+defmodule ActionDataFetcher.GPO.Parser.Worker do
+  use GenServer
+  import SweetXml
+
+  ## Worker API
+
+  def start_link(stuff) do
+    GenServer.start_link(__MODULE__, stuff)
+  end
+  
+  def handle_call({:parse_bill, {:filepath, bill_data_xml_path}}, _from, state) do
```

Comment:
none. this is a questionable part of GenServers tbh. I put state to future-proof, but perhaps this is bad form?